db/seeds.rb

```diff
@@ -1,7 +1,9 @@
+require_relative '../lib/populator_fix.rb'
+
```

Comment:
- [ ] It would be better if we improve our PR and make it more descriptive and professional. We can achieve it be an introduction and a list of formatted points or functionalities. Also, let us improve our README file by adding these commands `rails db:create db:migrate db:seed` before running the server. Otherwise, we will encounter errors

Reply:
I'm currently not using faker and populator gems.